nixos/modules/services/matrix/synapse.nix

```diff
@@ -707,6 +752,9 @@ in {
 
     services.matrix-synapse.configFile = configFile;
 
+    # default them, so they are additive
+    services.matrix-synapse.settings.extras = lib.mkDefault defaultExtras;
```

Comment:
These would be additive when using `services.matrix-synapse.settings.extras = lib.mkDefault [ "my" "extras" ];`, when omitting this `mkDefault` here, only `my` & `extras` would be set.

Iow the `mkDefault` isn't needed here.

Reply:
Dropped.